### ISSUE TITLE
[Issue]: LiveTV art is always goes back to default

### ISSUE THREAD

**Issue by iChristGit** [jellyfin/jellyfin#11775]
### Please describe your bug

![image](https://github.com/jellyfin/jellyfin/assets/20077386/98ffb3c8-c844-4cd8-95bc-196caec6f714)
I have a costum livetv art to show instead of the above default blue box, and whenever I change it it works once and then goes back to the deault blue color.
All other (TV shows/ movies / books/ music) works fine and not changing, but only LiveTV does that.

### Reproduction Steps

1.Latest version
2.Set a custom image as "Primary" for LiveTV
3.Quit the Media player and come back
4.No fancy pictures, only the default blue

### Jellyfin Version

10.9.0

### if other:

10.9.2

### Environment

```markdown
- OS: Win11
- Linux Kernel:N
- Virtualization:N
- Clients: Jelly Media Player
- Browser: Chrome / Edge
- FFmpeg Version: Default
- Playback Method: ?
- Hardware Acceleration: Enabled
- GPU Model: 3090 Ti
- Plugins: None
- Reverse Proxy: No
- Base URL:N
- Networking:N
- Storage:N
```

### Jellyfin logs

```shell
No logs
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by iChristGit** [comment]
> I can report the same issue. Reproduction steps same as OP.
> 
> Jellyfin Version 10.9.2
> 
> The png is stored in 'Jellyfin\Data\metadata\views\livetv\folder.png' but does not display in any of the apps or through the browser.
> 
> Log does not show anything on image change, or on reload with no cache.

I found out that going into Settings > Dashboard and then going back to the homescreen, the images shows, can you confirm it also your case?

**Comment by iChristGit** [comment]
Fixed for me with with latest jellyfin-web 10.9.6
Just waiting for Jellyfin Media Player to update.
Closing this issue.
